mantle/platform/conf/conf.go

```diff
@@ -1089,6 +1089,22 @@ func (c *Conf) AddAutoLogin() {
 	c.AddSystemdUnitDropin("serial-getty@.service", "10-autologin.conf", getAutologinUnit("serial-getty@.service", "--keep-baud 115200,38400,9600"))
 }
 
+// AddAutoResize adds an Ignition config for a `resize` function to resize serial consoles
+func (c *Conf) AddAutoResize() {
+	c.AddFile("/etc/profile.d/autoresize.sh", `
+# adapted from https://wiki.archlinux.org/title/Working_with_the_serial_console#Resizing_a_terminal
+resize_terminal() {
+	local IFS='[;' escape geometry x y
+	echo -en '\e7\e[r\e[999;999H\e[6n\e8'
+	read -sd R escape geometry
+	x=${geometry##*;} y=${geometry%%;*}
+	if [[ ${COLUMNS} -ne ${x} || ${LINES} -ne ${y} ]];then
+		stty cols ${x} rows ${y}
+	fi
+}
+PROMPT_COMMAND+=(resize_terminal)`, 0644)
```

Comment:
not sure if I like it running every time a new prompt is written to the screen. Should we just make it a callable function?

Also wish we could somehow get this functionality into `util-linux` or `coreutils`.

Reply:
> not sure if I like it running every time a new prompt is written to the screen. 

^^ that's a weakly held opinion. Definitely interested in what other people think.

Reply:
I see where you're coming from. OTOH, devshell mode is optimized for convenience and UX and I think doing it automatically fits into that picture well.